src/js/pages/universe/PackagesTab.js

```diff
@@ -30,6 +33,18 @@ const METHODS_TO_BIND = [
   'handleSearchStringChange'
 ];
 
+const shouldRenderUniverseOption = Hooks.applyFilter(
+  'hasCapability', false, 'packageAPI'
+);
+
+if (shouldRenderUniverseOption) {
+  MountService.MountService.registerComponent(
```

Comment:
@nLight I understand your concern and I do agree it would make sense. However, I started implementing your suggestion and then ran into a road block: how do we initialize this new plugin? Right now the only way to initialize plugins is to add them to the `ui-config.json` (or hard-code an `import` somewhere else in the app). I chatted with @rcorral about this and since we are moving away from the "plugins" approach, we didn't think it made sense to create a plugin for this until we can move toward the new "package" approach.

What do you think about this decision?